code/DocumentationParser.php

```diff
@@ -482,6 +482,9 @@ public static function rewrite_relative_links($md, $page)
                         $fileBaseLink,
                         $url
                     );
+                } else if (preg_match('/^#/', $url)) {
+                    // for relative links begining with a hash use the current page link
+                    $relativeUrl = Controller::join_links($baselink, $page->getRelativeLink(), '/', $url);
```

Comment:
`join_links` adds the `/` between params so the penultimate argument is not needed